server/backend/mlx.py

```
@@ -269,36 +294,62 @@ def count_tokens(text: str) -> int:
+    if isinstance(request.input, str):
+        user_input_content = request.input
+    else:
+        for item in request.input:
+            match item:
+                case UserMessageItemParam():
+                    _user_msg_item = item
+                    user_input_content = item.content.root  # pyright: ignore
+                case DeveloperMessageItemParam():
+                    dev_msg_item = item
+                case _:
+                    raise TypeError("unknown type")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)